ddtrace/internal/constants.py

```diff
@@ -0,0 +1,18 @@
+from ddtrace.constants import ANALYTICS_SAMPLE_RATE_KEY
+
+
+FILTERS_KEY = "FILTERS"
+SAMPLE_RATE_METRIC_KEY = "_sample_rate"
+SAMPLING_AGENT_DECISION = "_dd.agent_psr"
+SAMPLING_RULE_DECISION = "_dd.rule_psr"
+SAMPLING_LIMIT_DECISION = "_dd.limit_psr"
+HOSTNAME_KEY = "_dd.hostname"
+ENV_KEY = "env"
+KEEP_SPANS_RATE_KEY = "_dd.tracer_kr"
+
+NUMERIC_TAGS = (ANALYTICS_SAMPLE_RATE_KEY,)
+
+MANUAL_DROP_KEY = "manual.drop"
```

Comment:
these are used publicly

Reply:
We're probably going to have to cross-reference the documentation repo and onboarding docs to make sure we get it right